pulp_smash/config.py

```diff
@@ -545,11 +556,19 @@ def load(cls, xdg_subdir=None, config_file=None):
                 DeprecationWarning,
             )
             loaded_config["hosts"] = loaded_config.pop("systems")
+
+        try:
+            timeout = loaded_config["general"]["timeout"]
+        except KeyError:
+            # For old versions of settings.json without the value of timeout
+            # defined.
+            timeout = 1800
+
```

Comment:
all the above try can be

```python
timeout = loaded_config.get("general", {}).get("timeout", 1800)
```
try to `get` a `general` dict from config, if doesn't exist return empty dict, then try to `get` the `timeout` if not defined use default `1800`

Reply:
@rochacbruno, I will update properly.